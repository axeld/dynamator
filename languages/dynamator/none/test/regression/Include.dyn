<dynamator language="none">
<prolog><!--RETAIN Starting comment RETAIN--></prolog>
  <include file="Prolog.dyn"/>
  <include file="TestPath.include"/>
  <tag tag="title">
    <content>New title</content>
  </tag>
  <include file="Epilog.dyn"/>
<epilog><!--RETAIN Ending comment RETAIN-->
</epilog>
</dynamator>


