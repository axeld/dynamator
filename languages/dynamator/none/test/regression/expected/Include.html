<!--RETAIN Starting comment RETAIN--><!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
    <!--RETAIN the included prolog RETAIN-->
  <html>
  <head>
    <title>New title</title>
  </head>
  <body>
      <p>The -I argument worked.
    x
      <p>Nested include worked.
    </body>
</html>

    <!--RETAIN the included epilog RETAIN-->
  
<!--RETAIN This is the end. RETAIN-->
<!--RETAIN Ending comment RETAIN-->
<!-- generated by Dynamator Sun Feb 15 17:23:24 CST 2004 -->
